apollo-router/src/snapshots/apollo_router__apollo_telemetry__test__fragment.snap

```diff
@@ -0,0 +1,20 @@
+---
+source: apollo-router/src/apollo_telemetry.rs
+assertion_line: 329
+expression: normalized
+
+---
+
+{
```

Comment:
I don't get why some of these snapshots have whitespace reduced and others do not, but presumably that's because this part of the PR isn't done.